packages/itwin/grouping-mapping-widget/src/widget/components/SortableHorizontalTile.tsx

```diff
@@ -46,7 +46,13 @@ const SortableHorizontalTile = ({ id, ...props }: SortableHorizontalTileProps) =
     >
       <HorizontalTile
         dragHandle={
-          <div className="gmw-drag-icon" style={{ cursor: "grab" }} title="Drag & Drop" {...listeners}>
+          <div 
```

Comment:
How difficult would it be to replace it with https://itwin.github.io/iTwinUI/react/?path=/story/utilities-icon--default?

Reply:
switched to Icon